HIP/hip-0000-FT-JSON-SCHEMA.md

```diff
@@ -0,0 +1,181 @@
+- lightLogo  - a logo for the token designed for light backgrounds. svg is recommended.
+- lightLogoType - if the lightLogo  is specified, its mime type
+- darkLogo  - a logo for the token designed for dark backgrounds. svg is recommended.
+- darkLogoType - if the darkLogo  is specified, its mime type
+- website address -the address at which additional information on the token or corresponding project
+- chat/discord address - the address at which the project's community holds discussion
+- whitepaper address - the address at which the project's whitepaper can be found
+- properties - container for arbitrary token attributes not specifically listed above
+
+Below is the human-readable schema 
+
+```
+{
+    "description": "description of token - RECOMMENDED - max. of 500 characters",
+    "smallestUnitName" : "the name of the smallest denomination of these tokens, if the name exists - OPTIONAL",
+    "smallestUnitSymbol" : "the nsymbol of the smallest denomination of these tokens, if the symbol exists - OPTIONAL",
+    "creator": "creator(s) - RECOMMENDED",
+    "creatorDID": "DID  - OPTIONAL ",
+    "admin": "admin(s) - OPTIONAL",
+    "lightLogo": "IPFS CID or path to the token's light background logo file - RECOMMENDED",
+    "lightLogotype": "mime type - i.e. image/jpeg - CONDITIONALLY OPTIONAL ",
+    "darkLogo": "IPFS CID or path to the token's dark background logo file - RECOMMENDED",
+    "darkLogotype": "mime type - i.e. image/jpeg - CONDITIONALLY OPTIONAL ",
+    "website": "link to website -  OPTIONAL", 
+    "discussion": "link to discussion/discord -  OPTIONAL", 
+    "whitepaper": "link to whitepaper -  OPTIONAL",
+    "properties": "arbitrary additional data relevant to the token - OPTIONAL"
+}
+```
+
+
+
+### Formatting Notes
+
+#### URI Formatting
+
+URI’s shall follow the following format: protocol://resource_location
+
+For resources that are on the world wide web, the standard http and https protocols are acceptable. Ie. http://www.example.org/image/file.jpg
+
+For resources that are on IPFS, the protocol must be ipfs:// and the resource location must be the cid of the file. I.e. ipfs://bafkreibwci24bt2xtqi23g35gfx63wj555u77lwl2t55ajbfjqomgefxce
+
+For resources that stored are on the hedera file service, the protocol is hedera:mainnet//0.0.123456
+
+A more complete list of URI’s can be found here: https://en.wikipedia.org/wiki/List_of_URI_schemes
+
+### Mime Formatting
+
+Mime formatting shall follow the following format: type/subtype
+
+As a rule, mime types are all lower case. However apps should be programmed to accept any case for robustness.
+
+A list of common mime types can be found here: https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/MIME_types/Common_types
+
+Note that mime types for directories are not uniformly defined. Some IPFS CIDs point to directories rather than files, so this type is useful. This standard shall define it using the format: text/directory 
```

Comment:
```suggestion
Note that mime types for directories are not uniformly defined. Some IPFS CIDs point to directories rather than files, so this type is useful. This standard shall define it using the format: `text/directory` 
```